explorer/src/components/JupiterSwapModal.tsx

```diff
@@ -0,0 +1,562 @@
+		});
+		return path;
+	}
+	const swapUserInputTokens = () => {
+		const swapInputToken = inputToken;
+		const swapOutputToken = outputToken;
+		setTokenValues(swapOutputToken, "pay");
+		setTokenValues(swapInputToken, "receive");
+	}
+
+	const onClickSwapBestRoute = async () => {
+		if (routes && wallet.signAllTransactions && wallet.signTransaction) {
+			const bestRoute = routes[0];
+			await exchange({
+				wallet: {
+					sendTransaction: wallet?.sendTransaction,
+					publicKey: wallet?.publicKey,
+					signAllTransactions: wallet.signAllTransactions,
+					signTransaction: wallet.signTransaction,
+				},
+				routeInfo: bestRoute,
+				onTransaction: async (txid: any) => {
+					await connection.confirmTransaction(txid);
+					return await connection.getTransaction(txid, {
+						commitment: "confirmed",
+					});
+				},
+			});
+			setInputBalance(0);
+			setRecieveTokenBalance(0);
+			setExchangeAmount(0);
+			setInputToken({
+				chainId: 0,
+				address: '',
+				symbol: '',
+				name: '',
+				decimals: 3,
+				logoURI: '',
+				tags: [''],
+			});
+			setOutputToken({
+				chainId: 0,
+				address: '',
+				symbol: '',
+				name: '',
+				decimals: 3,
+				logoURI: '',
+				tags: [''],
+			});
+			setDisplayRoutes([]);
+			setBalance(0);
+		}
+	}
+	return (
+		<Modal {...props} centered>
+			<Modal.Body>
+				<div className="jupiter-modal">
+					<div className="d-flex mb-3">
+						{aggregator ? (
+							<div className="d-flex justify-content-between align-items-center p-3 me-4 w-100 selected-aggregator rounded-3"
+								onClick={() => setShowAggregatorSearch(true)}>
+								<div className="w-100 d-flex align-items-center">
+									<img className="me-3" src={aggregator.logo} alt={aggregator.name} width="25"></img>
+									<div><span>{aggregator.name}</span></div>
+								</div>
+								<div>
+									<span className="fe fe-align-justify" style={{ fontSize: "16px" }}></span>
+								</div>
+							</div>
+						) : (
+							<div
+								className="d-flex justify-content-between align-items-center p-3 me-4 w-100"
+								onClick={() => setShowAggregatorSearch(true)}>
+								<input className="form-control" placeholder="Select Aggregator"></input>
+								<div className="border-left">
+									<span className="fe fe-search ps-3"></span>
+								</div>
+							</div>)}
+						<button
+							className="d-flex align-items-center slippage-setting-btn p-3 rounded-3 opacity-text"
+							onClick={() => setShowSlippageSettings(true)}>
+							<div className="fe fe-sliders me-2"></div>
+							<div>{formValue.slippage}%</div>
+						</button>
+					</div>
+
+					{showAggregatorSearch ? (
+						<AggregatorSearchModal
+							aggregatorOptions={aggregatorOptions}
+							setAggregator={setAggregator}
+							aggregatorSearch={aggregatorSearch}
+							onAggregatorSearchChange={onAggregatorSearchChange}
+							setShowAggregatorSearch={setShowAggregatorSearch}
+						/>
+					) : null}
```

Comment:
One neat way to do this is to use && operator.
```suggestion
					{showAggregatorSearch && (
						<AggregatorSearchModal
							aggregatorOptions={aggregatorOptions}
							setAggregator={setAggregator}
							aggregatorSearch={aggregatorSearch}
							onAggregatorSearchChange={onAggregatorSearchChange}
							setShowAggregatorSearch={setShowAggregatorSearch}
						/>
					)}
```